README.rst

```diff
@@ -110,8 +110,9 @@ regularly and useful to understand ``atlas`` at a glance.
 
     Commands:
       pull      pull
-      -h, --help                  
-          --version               
```

Comment:
There is a ton of whitespace after both `--help` and `--version`. This PR removes it.